### PR TITLE
Update mad_strings.xml

### PR DESCRIPTION
**Pull request by Lenovoved**

### CODE REVIEW

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 1]
```diff
@@ -636,7 +636,7 @@
     <string name="pa_pie_settings_panel_summary">Включить кнопку показа открытой панели быстрых настроек</string>
     <string name="pa_pie_screenshot_title">Снимок экрана</string>
     <string name="pa_pie_screenshot_summary">Включить кнопку скнимка экрана</string>
-    <string name="pa_pie_color_title">Настройки цвета помощьника</string>
+    <string name="pa_pie_color_title">Настройки цвета помощника</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: так, давайте обойдёмся без Помощника. Это - Пай или Pie. Оставим его в исходном варианте.

**Review by iMiKED — CHANGES_REQUESTED** [review]
Please, don't accept those changes.
Pie is wrong translated as Helper or Assistant.
I think - Pie shouldn't be translated to Russian.
Please, don't accept those changes.

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 2]
```diff
@@ -540,10 +540,10 @@
     <string name="show_network_traffic_all">Исходящий и входящий</string>
     <string name="show_network_traffic_bit">Бит/сек.</string>
     <string name="show_network_traffic_byte">Байт/сек.</string>
-    <string name="show_network_traffic_500">500 мсек</string>
-    <string name="show_network_traffic_1000">1000 мсек</string>
-    <string name="show_network_traffic_1500">1500 мсек</string>
-    <string name="show_network_traffic_2000">2000 мсек</string>
+    <string name="show_network_traffic_500">500 м/сек</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml:543]: wrong!
Why meters/sec??? It is milliseconds!!!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 3]
```diff
@@ -540,10 +540,10 @@
...
+    <string name="show_network_traffic_1000">1000 м/сек</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml:544]: wrong!
Why meters/sec??? It is milliseconds!!!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 4]
```diff
@@ -540,10 +540,10 @@
...
+    <string name="show_network_traffic_1500">1500 м/сек</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml:545]: wrong!
Why meters/sec??? It is milliseconds!!!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 5]
```diff
@@ -540,10 +540,10 @@
...
+    <string name="show_network_traffic_2000">2000 м/сек</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml:546]: wrong!
Why meters/sec??? It is milliseconds!!!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 6]
```diff
@@ -586,43 +586,43 @@
     <string name="reset_android">Android</string>
 
     <!-- PA Pie control -->
-    <string name="pa_piecontrol_settings_summary">Настройки стиля, размера, цвета и целей Pie</string>
-    <string name="pa_pie_control_title">Настроить Pie</string>
-    <string name="pa_pie_control_enable_title">Включить управление Pie</string>
-    <string name="pa_pie_targets_title">Цели Pie</string>
-    <string name="pa_pie_style_title">Стиль Pie</string>
+    <string name="pa_piecontrol_settings_summary">Настройки стиля помощьника нажатий</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 7]
```diff
@@ -586,43 +586,43 @@
...
+    <string name="pa_pie_control_title">Настроить помощьника нажатий</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 8]
```diff
@@ -586,43 +586,43 @@
...
+    <string name="pa_pie_control_enable_title">Включить помощьника нажатий</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 9]
```diff
@@ -586,43 +586,43 @@
...
+    <string name="pa_pie_targets_title">Функции помощьника</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 10]
```diff
@@ -586,43 +586,43 @@
...
+    <string name="pa_pie_style_title">Внешний вид помощьника</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 11]
```diff
@@ -586,43 +586,43 @@
...
     <string name="pa_pie_menu_title">Всегда показывать кнопку меню</string>
     <string name="pa_pie_menu_summary">Всегда показывать кнопку меню, даже если она не нужна</string>
-    <string name="pa_pie_size_title">Размер Pie</string>
-    <string name="pa_pie_size_summary">Размер элементов управления Pie</string>
+    <string name="pa_pie_size_title">Размер помощьника</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 12]
```diff
@@ -586,43 +586,43 @@
...
+    <string name="pa_pie_size_summary">Размер элементов управления</string>
     <string name="pa_pie_size_small">Маленький</string>
     <string name="pa_pie_size_default">@string/pa_pie_mode_default</string>
-    <string name="pa_pie_size_large">Большой</string>
-    <string name="pa_pie_size_gigantic">Гигантский</string>
-    <string name="pa_pie_size_monster">Чудовищный</string>
+    <string name="pa_pie_size_large">Средний</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml:600]: Wrong translation!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 13]
```diff
@@ -586,43 +586,43 @@
...
+    <string name="pa_pie_size_gigantic">Большой</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml:601]: Wrong translation!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 14]
```diff
@@ -586,43 +586,43 @@
...
+    <string name="pa_pie_size_monster">Очень большой</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml:602]: Wrong translation!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 15]
```diff
@@ -586,43 +586,43 @@
...
     <string name="pa_pie_size_enormous">Огромный</string>
     <string name="pa_pie_mode_default">По умолчанию</string>
-    <string name="pa_pie_gravity_title">Расположение Pie</string>
-    <string name="pa_pie_gravity_summary">Выбор расположения Pie</string>
-    <string name="pa_pie_gravity_bottom">Внизу</string>
+    <string name="pa_pie_gravity_title">Расположение помощьника</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 16]
```diff
@@ -586,43 +586,43 @@
...
+    <string name="pa_pie_gravity_bottom">Снизу</string>
     <string name="pa_pie_gravity_right">Справа</string>
     <string name="pa_pie_gravity_left">Слева</string>
     <string name="pa_pie_gap_title">Расстояние между элементами</string>
-    <string name="pa_pie_gap_summary">Настройка расстояния, отделяющего элементы Pie друг от друга</string>
-    <string name="pa_pie_angle_title">Угол Pie</string>
-    <string name="pa_pie_angle_summary">Настроить угол краёв Pie</string>
-    <string name="pa_pie_angle_none">Нет (старый)</string>
+    <string name="pa_pie_gap_summary">Настройка расстояния между элементами управления</string>
+    <string name="pa_pie_angle_title">Углы помощьника</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 17]
```diff
@@ -586,43 +586,43 @@
...
+    <string name="pa_pie_angle_summary">Настроить угол краёв элементов</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 18]
```diff
@@ -586,43 +586,43 @@
...
+    <string name="pa_pie_angle_big">По умолчанию</string>
     <string name="pa_pie_angle_huge">Большой</string>
     <string name="pa_pie_angle_super">Огромный</string>
     <string name="pa_pie_gap_none">Нет</string>
     <string name="pa_pie_gap_small">Маленькое</string>
-    <string name="pa_pie_gap_big">Большое (по умолчанию)</string>
+    <string name="pa_pie_gap_big">По умолчанию</string>
     <string name="pa_pie_gap_huge">Большое</string>
-    <string name="pa_pie_mode_title">Стиль Pie</string>
+    <string name="pa_pie_mode_title">Стиль помощьника</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml:623]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 19]
```diff
@@ -586,43 +586,43 @@
...
+    <string name="pa_pie_angle_none">Нет</string>
     <string name="pa_pie_angle_small">Маленький</string>
-    <string name="pa_pie_angle_big">Нормальный (по умолчанию)</string>
+    <string name="pa_pie_angle_big">По умолчанию</string>
     <string name="pa_pie_angle_huge">Большой</string>
     <string name="pa_pie_angle_super">Огромный</string>
     <string name="pa_pie_gap_none">Нет</string>
     <string name="pa_pie_gap_small">Маленькое</string>
-    <string name="pa_pie_gap_big">Большое (по умолчанию)</string>
+    <string name="pa_pie_gap_big">По умолчанию</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml:621]: You should translate 'Big'. Wrong translation.

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 20]
```diff
@@ -636,8 +636,8 @@
     <string name="pa_pie_settings_panel_summary">Включить кнопку показа открытой панели быстрых настроек</string>
     <string name="pa_pie_screenshot_title">Снимок экрана</string>
     <string name="pa_pie_screenshot_summary">Включить кнопку скнимка экрана</string>
-    <string name="pa_pie_color_title">Настройки цвета Pie</string>
-    <string name="pa_pie_color_style_title">Стиль цвета Pie</string>
+    <string name="pa_pie_color_title">Настройки цвета помощьника</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 21]
```diff
@@ -636,8 +636,8 @@
...
+    <string name="pa_pie_color_style_title">Стиль цвета помощьника</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 22]
```diff
@@ -647,6 +647,6 @@
     <string name="pa_pie_juice_title">Цвет батареи</string>
     <string name="pa_pie_button_color">Цвет кнопки</string>
     <string name="pa_pie_button_color_title">Цвет кнопок</string>
-    <string name="pa_pie_always_right_title">Pie всегда справа</string>
-    <string name="pa_pie_always_right_summary">Pie всегда быдет выводиться справа в горизонтальной ориентации</string>
-</resources>
\ No newline at end of file
+    <string name="pa_pie_always_right_title">Помощьник всегда справа</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 23]
```diff
@@ -647,6 +647,6 @@
...
+    <string name="pa_pie_always_right_summary">Помощьник всегда быдет выводиться справа в горизонтальной ориентации</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Check grammar and spell. Wrong translation.

**Review by iMiKED — CHANGES_REQUESTED** [review]
Many errors in translation.
Pie shouldn't be translated into Russian!
Please, don't accept it!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 24]
```diff
@@ -586,14 +586,14 @@
     <string name="reset_android">Android</string>
 
     <!-- PA Pie control -->
-    <string name="pa_piecontrol_settings_summary">Настройки стиля помощьника нажатий</string>
-    <string name="pa_pie_control_title">Настроить помощьника нажатий</string>
-    <string name="pa_pie_control_enable_title">Включить помощьника нажатий</string>
-    <string name="pa_pie_targets_title">Функции помощьника</string>
-    <string name="pa_pie_style_title">Внешний вид помощьника</string>
+    <string name="pa_piecontrol_settings_summary">Настройки стиля помощника нажатий</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 25]
```diff
@@ -586,14 +586,14 @@
...
+    <string name="pa_pie_control_title">Настроить помощника нажатий</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 26]
```diff
@@ -586,14 +586,14 @@
...
+    <string name="pa_pie_control_enable_title">Включить помощника нажатий</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 27]
```diff
@@ -586,14 +586,14 @@
...
+    <string name="pa_pie_targets_title">Функции помощника</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml]: Pie shouldn't be translated into Russian!

File: packages/apps/Settings/res/values-ru/mad_strings.xml [hunk 28]
```diff
@@ -586,14 +586,14 @@
...
+    <string name="pa_pie_style_title">Внешний вид помощника</string>
```
Comment (iMiKED) [inline packages/apps/Settings/res/values-ru/mad_strings.xml:593]: Pie shouldn't be translated into Russian!

**Review by iMiKED — CHANGES_REQUESTED** [review]
Pie shouldn't be translated into Russian!
Many errors.
Please, don't accept this translation.

File: packages/apps/Settings/res/values-ru/mad_strings.xml (hunk 1 above)
Comment (Lenovoved) [reply packages/apps/Settings/res/values-ru/mad_strings.xml]: Поучи ещё! Что за пирог?! Ты в своём уме? Не крути мозги и пусть перевод несёт полный смысл а не отрезки пирогов.

Comment (iMiKED) [reply packages/apps/Settings/res/values-ru/mad_strings.xml]: Это - PIE! От аналогии с пирогом.
Поэтому он не нуждается в переводе, нет нормального перевода для него, а переводить как "Помощник нажатий" это полный бред!

**Comment by dragonpt** [comment]
Thanks for having time, but I have to refuse this merges.

Thx

**Comment by dragonpt** [comment]
I have asked an Russian friend that also agrees, this are wrongly / not on the right sense translated due to the context where those are used.

Thanks anyway ;-)

**Comment by iMiKED** [comment]
Thank you so much.

**Pull request closed without merging**